src/home_robot/home_robot/utils/image.py

```diff
@@ -159,6 +159,7 @@ def to_o3d_point_cloud(xyz, rgb=None, mask=None):
 
 def show_point_cloud(xyz, rgb=None, orig=None, R=None):
     # http://www.open3d.org/docs/0.9.0/tutorial/Basic/working_with_numpy.html
+    import open3d as o3d
```

Comment:
can you add an issue (tag me) to remove this entirely? it should be done in my new PR https://github.com/facebookresearch/home-robot/pull/58

we also have a duplicate point_cloud.py in utils

Reply:
@cpaxton - `import open3d as o3d` can fail on some older CPUs. It's worth making this import optional. This is not handled in https://github.com/facebookresearch/home-robot/pull/58 yet.

Reply:
interesting - why is that?
i think we should only import open3d in point cloud specific code (like point_cloud.py)